ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -581,8 +581,8 @@ Result conv2d_DRAM(
         compute_config,
         device);
 
-    ttnn::operations::op_slicing::run_sliced_op(
-        input_tensor_on_device, dram_output_tensor, &slice_attr, dram_slice_config_);
+    std::vector<std::reference_wrapper<Tensor>> output_tensors = {std::ref(dram_output_tensor)};
```

Comment:
why do we need this chagne here in conv2d?

Reply:
well we didn't need this so far right and this change is for pools, I'm trying to understand why need these ref tensors now